.github/workflows/phpstan.yml

```
@@ -34,6 +34,9 @@ jobs:
       # @link https://github.com/marketplace/actions/install-composer-dependencies
       - name: Install Composer dependencies
         uses: "ramsey/composer-install@v2"
+        with:
+          # Bust the cache at least once a month - output format: YYYY-MM-DD.
+          custom-cache-suffix: $(date -u -d "-0 month -$(($(date +%d)-1)) days" "+%F")
```

Could this be `date +%Y-%m`?
```suggestion
          custom-cache-suffix: $(date -u "+%Y-%m")
```

---

You're 100% right. Looks like I updated this everywhere else where I proposed this change, but missed it here as the PR still hasn't been merged (so it didn't show up on my system-wide code search as I was in another branch).